presidio-anonymizer/tests/anonymizers/test_encrypt.py

```diff
@@ -0,0 +1,31 @@
+from unittest import mock
+
+import pytest
+
+from presidio_anonymizer.anonymizers.encrypt import Encrypt
+from presidio_anonymizer.entities import InvalidParamException
+from presidio_anonymizer.services.aes_cipher import AESCipher
+
+
+@mock.patch.object(AESCipher, "encrypt")
+def test_when_anonymize_then_aes_encrypt_called_and_its_result_is_returned(
```

Comment:
Nit should be:
test_give_..._then_...
For all tests...

Reply:
Done

Reply:
Not all test names were changed :)